snap/local/core_launcher.sh

```diff
@@ -7,5 +7,5 @@ TURTLEBOT3_MODEL="$(snapctl get turtlebot3-model)"
 export LDS_MODEL
 export TURTLEBOT3_MODEL
 
-${SNAP}/opt/ros/noetic/bin/roslaunch turtlebot3c_bringup turtlebot3c_bringup.launch simulation:=$SIMULATION
+${SNAP}/roslaunch turtlebot3c_bringup turtlebot3c_bringup.launch simulation:=$SIMULATION
```

Comment:
You shouldn't even need the `$SNAP/` prefix in these script.

Reply:
So I tested since I thought I remembered something weird, and indeed there is something. While here in the core_launcher.sh I could remove the $SNAP it's not the case of all the scripts. The scripts `mux_select_*` in the command chain, when I remove the $SNAP prefix I get:
```
/snap/turtlebot3c/x2/usr/bin/mux_select_key_vel.sh: line 2: opt/ros/noetic/bin/rosservice: No s
uch file or directory
```
I am not sure if this is a bug. I am concerned that it might be confusing for users. I would prefer to keep the "$SNAP" in anything that is outside the `snapcraft.yaml` before we know more if it's a bug or not.